lib/plugins/create/templates/aws-nodejs-docker/serverless.yml

```diff
@@ -0,0 +1,119 @@
+# Welcome to Serverless!
+#
+# This file is the main config file for your service.
+# It's very minimal at this point and uses default values.
+# You can always add more config options for more control.
+# We've included some commented out config examples here.
+# Just uncomment any of them to get that config option.
+#
+# For full config options, check the docs:
+#    docs.serverless.com
+#
+# Happy Coding!
+
+service: test-aws-nodejs-docker # NOTE: update this with your service name
+# app and org for use with dashboard.serverless.com
+#app: your-app-name
+#org: your-org-name
+
+# You can pin your service to only deploy with a specific Serverless version
+# Check out our docs for more details
+frameworkVersion: '2'
+
+provider:
+  name: aws
+  lambdaHashingVersion: 20201221
+  ecr:
+    # In this section you can define images that will be built locally and uploaded to ECR
+    images:
+      appimage:
+        path: ./
+  apiGateway:
+    shouldStartNameWithService: true
+
+# you can overwrite defaults here
```

Comment:
What is your opinion on these commented out settings? Should we keep these for templates or remove them?

Reply:
I would probably keep same comments as we have in regular nodejs tremplate

Reply:
:+1: I'm keeping these then